### DISCUSSION TITLE
Dark Mode toggle and OS preference

### DISCUSSION THREAD

**Discussion by LinThitHtwe** [tailwindlabs/tailwindcss#13712] [Help]
Hello ? Is there a way to enable dark mode/ light mode in tailwind class but also kept OS preference when user first visit to the website?
I know that we can toggle dark mode/light mode by adding dark class to DOM, but is there a way to also use default OS preference when user visit the web for the first time ? 
```javascript

import type { Config } from "tailwindcss";

const config: Config = {
  darkMode: "class",
};
export default config;

```

**Comment by wongjn** [comment]
Yep! See [the documentation](https://tailwindcss.com/docs/dark-mode#supporting-system-preference-and-manual-selection) for an example.
